scipy/sparse/linalg/isolve/minres.py

```diff
@@ -1,7 +1,7 @@
 from numpy import sqrt, inner, zeros, inf, finfo
 from numpy.linalg import norm
 
-from .utils import make_system
+from scipy.sparse.linalg.isolve.utils import make_system
```

Comment:
This was to import directly from the file on the relative path, so you don't need it.

Reply:
Hi @ilayn thank you I'll do the change.